docs/metrics/prometheus/grafana/minio-overview.json

```diff
@@ -68,10 +68,10 @@
   },
   "description": "MinIO dashboard - https://min.io/",
   "editable": true,
-  "gnetId": 13502,
+  "gnetId": 12617,
```

Comment:
I am confused about which dashboard is supposed to be deployed?  If I import either one they both have the same name and UID, and the 13502 one has the official minio name.

Reply:
yeah changed it to `13502`. That is the correct ID